recipes/bioconductor-pmp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.3.2" %}
+{% set name = "pmp" %}
+{% set bioc = "3.13" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: '{{ version }}'
+source:
+  url:
+    - 'https://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
+  md5: 419d34dc63dbd48ec7481cc3316a429c
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+# Suggests: testthat, covr, knitr, rmarkdown, BiocStyle, gridExtra, magick
+requirements:
+  host:
+    - 'bioconductor-impute >=1.64.0,<1.65.0'
```

Comment:
I dont think we need to be so specific here? bioc 3.13 will ensure the correct version is installed? Or do we need to specify the version that corresponds to bioc 3.13?

Reply:
After discussing this we decided to stick with the specified version. We will use this conda build within bioc 3.12 but use the source from 3.13.